core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -43,7 +43,7 @@ oppia.controller('StateHints', [
 
       $scope.activeHintIndex = null;
     });
-
+    
```

Comment:
Remove extra spaces. (In general, check your changes in the ["Files Changed" tab](https://github.com/oppia/oppia/pull/4316/files) first before finalizing the PR.)

Reply:
oops missed this when i deleted lines for the last commit. Sorry. Done now.